app/src/main/java/framgia/com/ichat/data/model/Message.java

```diff
@@ -66,4 +66,10 @@ public String getSenderImage() {
     public void setSenderImage(String senderImage) {
         mSenderImage = senderImage;
     }
+    public class MessageKey{
+        public static final String CONTENT_DEFAULT = "Invite people join chat with you!";
+        public static final String SENDER_ID_DEFAULT = "3a6Fo5rrUcBqhUJcLsP0";
```

Comment:
cái này là id gì vậy e nhỉ

Reply:
@Tuanlvt 
cái đoạn id thì mục đích của em là sẽ tao ra 1 message defaut thông báo cho user để có thể add thêm người vào ạ
còn cái đoạn check trong positionAdapter :
Thì e đọc tài liệu thì tài liệu nói như vầy ạ: Check if an item was deleted, but the user clicked it before the UI removed it